mock-thunderhead/src/main/java/com/sequenceiq/thunderhead/grpc/service/auth/MockUserManagementService.java

```diff
@@ -381,7 +382,10 @@
     private boolean ephemeralDisksForTempDataEnabled;
 
     @Value("${auth.mock.datalake.multiaz.enable}")
-    private boolean multiAzDataLakeEnabled;
+    private boolean enableMultiAzDataLake;
```

Comment:
Lajos suggest this or are you forget to rename the field?

Reply:
Lajos mentioned to use enableMultiAzFreeIpa for ipa I thought it is better to make it consistent for datalake